packages/core/test/controls/numericInputTests.tsx

```diff
@@ -212,6 +218,410 @@ describe("<NumericInput>", () => {
         });
     });
 
+    describe("Keyboard text entry in input field", () => {
+
```

Comment:
It's a little tricky, since certain keys hop from "invalid" lists into "valid" lists depending on the prop flag. I'll see what I can do though.